src/sortStudents.ts

```diff
@@ -1,16 +1,70 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAverageGrade(grades: number[]): number {
+  return grades
+    .reduce((sum: number, current: number) => sum + current, 0)
+    / grades.length;
```

Comment:
would be easier to read this way
```suggestion
  return grades.reduce((sum: number, current: number) => (
    sum + current
  ), 0)/ grades.length;
```